xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/Game.h

```diff
@@ -947,6 +947,126 @@ class ATTR_DLL_LOCAL CInstanceGame : public IAddonInstance
   {
     return GAME_ERROR_NOT_IMPLEMENTED;
   }
+
+  //============================================================================
+  /// @brief Generates a RetroAchievements hash for a given game that
+  ///        that can be used to identify the game by RetroAchievements
+  ///
+  /// @param[out] hash The hash of the file. It's size must be >=33 characters
+  /// @param[in] consoleID The console ID as it is defined by rcheevos for
+  ///                      the console the rom is made for
+  /// @param[in] filePath The path of the rom
```

Comment:
I prefer capital `ROM` because abbreviations are capitalized. But it's up to you.